curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/66507c5f5766304f531cdfea.md

```diff
@@ -0,0 +1,90 @@
+---
+id: 66507c5f5766304f531cdfea
+title: Step 50
+challengeType: 1
+dashedName: step-50
+---
+
+# --description--
+
+Learning about functions for the first time can sometimes be a lot to take in. Before you move onto the next lessons, you will review what you have learned so far about about functions.
+
+In this step, you will need to do the following:
+
+- Create a function called ```addTwoNumbers``` that takes two parameters called ```num1``` and ```num2```.
```

Comment:
```suggestion
- Create a function called `addTwoNumbers` that takes two parameters called `num1` and `num2`.
```

Inline code takes one backtick, not three. This will need to be fixed throughout your changes.